docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -85,6 +85,56 @@ There you can find links to upgrade notes for other versions too.
             </exec>
         </target>
     ```
+- *(low priority)* use redis as cache for doctrine and framework ([#930](https://github.com/shopsys/shopsys/pull/928))
+    - update `app/config/packages/framework.yml`:
+    ```diff
+    framework:
+    +    annotations:
+    +        cache: shopsys.framework.cache_driver.annotations_cache
+    ```
+    - update `app/config/packages/snc_redis.yml`:
+    ```diff
+    snc_redis:
+        clients:
+            ...
+    +       framework_annotations:
+    +           type: 'phpredis'
+    +           alias: 'framework_annotations'
+    +           dsn: 'redis://%redis_host%'
+    +           options:
+    +               prefix: '%env(REDIS_PREFIX)%%build-version%:cache:framework:annotations:'
+    ```
+    - update `app/config/packages/doctrine.yml`:
```

Comment:
what about instructions for production ?
will be distribution of cache done in current deployment process within already implemented phing targets?

Reply:
You do not need to do anything else after this update